proxy/gateway.go

```diff
@@ -115,6 +133,17 @@ func (gw *Gateway) Handle(w http.ResponseWriter, r *http.Request) {
 	})
 	r = r.WithContext(ctx)
 
+	if gw.tracer != nil {
+		tracingCtx, span := gw.tracer.Start(r.Context(), "proxy.handle", trace.WithSpanKind(trace.SpanKindServer))
+		span.SetAttributes(attribute.String("router.id", source.ID))
+		span.SetAttributes(attribute.String("router.name", utils.PointerValue(source.Name)))
+		span.SetAttributes(attribute.String("router.workspaceId", source.WorkspaceId))
+		span.SetAttributes(attribute.String("router.async", fmt.Sprint(source.Async)))
+		span.SetAttributes(semconv.HTTPRoute(source.Path))
```

Comment:
prefix `router.` needs to be changed to something else. WebhookX doesn't have router concept.